### ISSUE TITLE
Ability to disable Zones

### ISSUE THREAD

**Issue by eliotstocker** [eliotstocker/Light-Controller#45]
Add the ability to disable/enable each zone seperately, sounds easy but requires a bit of thinking on the pager etc

**Comment by dudeofawesome** [comment]
I've started work on this on my [`feature-disable-zones`](https://github.com/dudeofawesome/Light-Controller/tree/feature-disable-zones) branch.

I've got it working great on the main activity of the phone app, and I think I've got it working on the Wear app too, but I'm having trouble testing that. It appears that preferences aren't syncing between the phone and the watch.

**Comment by eliotstocker** [comment]
sounds good :) not sure how we would do this with widgets, maybe just show less in the widget itself

Im going to set up a slack group where we can discuss issues such as the wear stuff, will post a link in a new issue for sign up

**Comment by mrwhale** [comment]
Yeah I'll have to merge this so I can play around with integrating into pebble too, nice work:) I did start some work on this a few weeks back but it never made it into my repo, still sitting locally

**Comment by dudeofawesome** [comment]
I've completed this feature, and opened a [pull request](https://github.com/eliotstocker/Light-Controller/pull/72) #72

**Comment by eliotstocker** [comment]
awsome!
